include/TCPServer.hpp

```diff
@@ -100,8 +109,12 @@ class TCPServer {
 
   int num_servers_;
 
-  // TODO(ljoswiak): Update to store multiple rounds
-  Round *round_;
+  // Mencius state.
+  std::unordered_map<int, TCPConnection::pointer> clients_;
+  std::unordered_map<int, std::shared_ptr<Round>> rounds_;
+  std::unordered_map<int, KVStore::AMOCommand> proposed_;
```

Comment:
Not necessary, but it might be helpful to have these as a "state" struct instead of 3 maps.

Reply:
Easier to keep it like this for now, but I agree, it is not ideal.